core/tests/test_utils.py

```diff
@@ -2590,23 +2561,19 @@ def get_pending_tasks(self, queue_name=None):
             list(Task). List of tasks in a single queue or in the entire
             taskqueue.
         """
-        return self.taskqueue_services_stub.get_pending_tasks(
+        return self._taskqueue_services_stub.get_pending_tasks(
             queue_name=queue_name)
 
     def count_jobs_in_mapreduce_taskqueue(self, queue_name):
         """Counts the jobs in the given mapreduce taskqueue."""
         return len(self.get_pending_mapreduce_tasks(queue_name=queue_name))
 
     def get_pending_mapreduce_tasks(self, queue_name=None):
-        """Returns the jobs in the given mapreduce taskqueue.
-        If queue_name is None, defaults to returning the jobs in all available
-        queues.
+        """Returns the jobs in the given mapreduce taskqueue. If queue_name is
+        None, defaults to returning the jobs in all available queues.
         """
-        if queue_name is not None:
-            return self.mapreduce_taskqueue_stub.get_filtered_tasks(
-                queue_names=[queue_name])
-        else:
-            return self.mapreduce_taskqueue_stub.get_filtered_tasks()
+        queue_names = queue_name and [queue_name]
```

Comment:
Don't do this, it's cryptic and needs to be unraveled. Try to be more explicit.

Reply:
Done.